lib/Cli.php

```diff
@@ -431,9 +418,10 @@ protected function convert($vObj)
         }
 
         if ($forceInput && $vObj->name !== $forceInput) {
-            throw new \Exception('You cannot convert a '.strtolower($vObj->name).' to '.$this->format);
+            throw new Exception('You cannot convert a '.strtolower($vObj->name).' to '.$this->format);
         }
         if ($convertVersion) {
+            /** @var VCard $vObj */
```

Comment:
Same here. Inline `@var` should be double checked..

Reply:
removed